source/_integrations/linear_garage_door.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: "Linear Garage Door"
```

Comment:
```suggestion
title: "Linear garage door"
```
We use sentence-style capitalization also for headings. Unless the product's actual name is spelled in caps (Linear Garage Door), I would lower case this.